.github/workflows/release.yml

```diff
@@ -40,7 +40,6 @@ jobs:
         with:
           tox-env: package
       - name: Determine Tag
-        id: determine-tag
```

Comment:
The release workflow uses `>> ${GITHUB_ENV}` + env vars to communicate between steps; so the id was not needed. That's only needed to refer to outputs as is done above in the run-tox action using the set-output workflow command.